### PR TITLE
feat: channel and integration identifer logging

### PR DESCRIPTION
**Pull request by grablack**
Log when a consumer views or interacts with a modal.

### CODE REVIEW

File: src/library/zoid/message/component.js [hunk 1]
```diff
@@ -214,7 +214,8 @@ export default createGlobalVariableGetter('__paypal_credit_message__', () =>
                                     type: 'message',
                                     messageRequestId,
                                     account: merchantId || account,
-                                    trackingDetails
+                                    trackingDetails,
+                                    channel: channel ?? 'UPSTREAM'
```
Comment (Seavenly) [inline src/library/zoid/message/component.js]: Can we add the channel prop to the props list like [the modal](https://github.com/paypal/paypal-messaging-components/blob/58df1de8d4c133bcd2590f57c351157c25b55c34/src/library/zoid/modal/component.js#L111)? And we should consider adding the default value by adding `default: () => 'UPSTREAM'` to the prop descriptors similar to [this](https://github.com/paypal/paypal-messaging-components/blob/58df1de8d4c133bcd2590f57c351157c25b55c34/src/library/zoid/modal/component.js#L302)?

File: src/library/zoid/modal/component.js [hunk 2]
```diff
@@ -243,7 +251,9 @@ export default createGlobalVariableGetter('__paypal_credit_modal__', () =>
                                     type: 'modal',
                                     messageRequestId,
                                     account: merchantId || account,
-                                    trackingDetails
+                                    trackingDetails,
+                                    channel: channel ?? 'UPSTREAM',
```
Comment (Seavenly) [inline src/library/zoid/modal/component.js]: Similar to the above comment. Consider using the default prop descriptor to handle this.

File: src/library/zoid/message/component.js (hunk 1 above)
Comment (grablack) [reply src/library/zoid/message/component.js]: updated

File: src/library/zoid/modal/component.js (hunk 2 above)
Comment (grablack) [reply src/library/zoid/modal/component.js]: updated

**Review by Seavenly — APPROVED** [review]

**Pull request merged**